content/spin/https-trigger.md

```diff
@@ -0,0 +1,55 @@
+title = "Exposing HTTP Triggers Using HTTPS"
+template = "spin_main"
+date = "2022-02-01T00:00:00Z"
+[extra]
+
+---
+- [HTTP vs HTTPS](#http-vs-https)
+- [Routing](#routing)
+- [Defining the HTTP Object](#defining-the-http-object)
+
+## HTTP vs HTTPS
+
+The [Spin HTTP trigger document](./http-trigger.md) provides a great overview of the HTTP trigger, as well as implementation details around the WebAssembly component model and how it
+is used in Spin. However, there may be certain situations where you would want to accept only HTTPS requests (and reject HTTP requests). This raises questions like, do I have to provide a TLS certificate, and so forth?
```

Comment:
It's not clear to me why we don't answer those questions in the HTTP article - why does the user need to discover and navigate to a separate page?